Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -293,6 +293,31 @@ def reactivate_user(self, username):
         self._get_sdk().users.reactivate(user_id)
         return user_id
 
+    def get_legal_hold_matter(self, matter_name):
+        if matter_name:
```

Comment:
if matter_name is not defined, it should return another error instead of `Code42LegalHoldMatterNotFoundError`. Otherwise it might raise another exception. `missing value....`. Moreover I wonder if this "if condition" is required, if `matter_name` is defined as a required parameter then it would fail early.

Reply:
This handles the scenario where an empty string is passed to `matter_name`, but I agree that the error should be different.